apps/vessel-history/features/profile/components/auth-icon/AuthIcon.module.css

```diff
@@ -0,0 +1,28 @@
+.authorized {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 50%;
+  margin-right: 0.3rem;
+  background: #fae9a0;
+}
+
+.false {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 50%;
+  margin-right: 0.3rem;
+  background: #d85676;
+  color: #fff;
+}
+
+.pending {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 50%;
+  margin-right: 0.3rem;
+  background: #afbbca;
+  color: #163f89;
+}
```

Comment:
For colors and margins use the corresponding variables

Reply:
I created new vars in ui-components, because the same colors not exist, CVP has the colors in the project